tests/sdk/go/switch-ssa-delete-container/step1/main.go

```diff
@@ -0,0 +1,96 @@
+package main
+
+import (
+	"github.com/pulumi/pulumi-kubernetes/sdk/v4/go/kubernetes"
+	appsv1 "github.com/pulumi/pulumi-kubernetes/sdk/v4/go/kubernetes/apps/v1"
+	corev1 "github.com/pulumi/pulumi-kubernetes/sdk/v4/go/kubernetes/core/v1"
+	metav1 "github.com/pulumi/pulumi-kubernetes/sdk/v4/go/kubernetes/meta/v1"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+)
+
+func main() {
+	pulumi.Run(func(ctx *pulumi.Context) error {
+		prov, err := kubernetes.NewProvider(ctx, "k8s", &kubernetes.ProviderArgs{
+			EnableServerSideApply: pulumi.BoolPtr(false),
+		})
+		if err != nil {
+			return err
+		}
+
+		_, err = appsv1.NewDeployment(ctx, "deployment", &appsv1.DeploymentArgs{
+			ApiVersion: pulumi.String("apps/v1"),
+			Kind:       pulumi.String("Deployment"),
+			Metadata: &metav1.ObjectMetaArgs{
+				Annotations: pulumi.StringMap{
+					"deployment.kubernetes.io/revision": pulumi.String("1"),
+				},
+				Name:      pulumi.String("nginx"),
+				Namespace: pulumi.String("default"),
```

Comment:
We should put the Deployment in a new Namespace to avoid possible resource conflicts between parallel tests on the shared CI cluster.